FirebaseAuth/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 - [changed] Added a `X-Firebase-GMPID` header to network requests.
+- [fixed] macOS Extension access to Shared Keychain by adding kSecUseDataProtectionKeychain recommended key (#8950, #6876)
```

Comment:
Based on #8950's title, the keychain prompt was mentioning `com.firebase.FIRInstallations.installations`.

Could you confirm the bundle ID(s) mentioned in the keychain prompts you are seeing (without this fix)?

Reply:
#8950 is a bit misleading because right now a keychain entry name is an API KEY, not a `com.firebase.FIRInstallations.installations`. Give me a few minutes, I'll find a commit that changed this.
So for me, a prompt was asking to access to `AlzaSy...` which is my app Firebase API KEY.

Reply:
I found it https://github.com/firebase/firebase-ios-sdk/pull/5603. It was a critical PR. Otherwise, every macOS application with Firebase Auth tries to use the same keychain entry to store app auth information.

Reply:
I see- thanks! I was wondering because I think this PR's solution may be applied to other parts of Firebase that use the keychain. Again, great investigation!

Reply:
You are right. It should be applied everywhere where `kSecAttrAccessGroup` is used, but `FIRAuthStoredUserManager` is the only place I found.